rotkehlchen/api/rest.py

```diff
@@ -1646,29 +1653,22 @@ def _import_history_debug(self, filepath: Union[FileStorage, Path]) -> Dict[str,
             """Imports the PnL debug data for processing and report generation"""
             json_importer = DebugHistoryImporter(self.rotkehlchen.data.db)
             if isinstance(filepath, Path):
-                success, data = json_importer.import_history_debug(filepath=filepath)
+                success, msg, events = json_importer.import_history_debug(filepath=filepath)
             else:
                 tmpfilepath = self.import_tmp_files[filepath]
-                success, data = json_importer.import_history_debug(filepath=tmpfilepath)
+                success, msg, events = json_importer.import_history_debug(filepath=tmpfilepath)
                 tmpfilepath.unlink(missing_ok=True)
                 del self.import_tmp_files[filepath]
 
             if success is False:
                 return wrap_in_fail_result(
-                    message=data['msg'],
+                    message=msg,
                     status_code=HTTPStatus.CONFLICT,
                 )
-            dbevents = DBHistoryEvents(self.rotkehlchen.data.db)
-            with self.rotkehlchen.data.db.conn.read_ctx() as cursor:
-                events = dbevents.get_history_events(
-                    cursor=cursor,
-                    filter_query=HistoryEventFilterQuery.make(),
-                    has_premium=True,
-                )
             self.rotkehlchen.accountant.process_history(
-                start_ts=data['data']['start_ts'],
-                end_ts=data['data']['end_ts'],
-                events=list(events),
+                start_ts=events[0].get_timestamp(),
```

Comment:
Hmm this should not be like that. Aren't we also exporting the settings of the PnL run? So the start and end timestamp? This is what should be used here.

End result would probably be the same, but it makes sense to have the same run exported by the user, to be ran while we debug.